ReportService/ReportTask/TaskWorker.cs

```diff
@@ -196,7 +196,7 @@ public void RunTask(IReportTaskRunContext taskContext)
             catch (Exception e)
             {
                 success = false;
-                var msg = $"Task {taskContext.TaskId} is not completed. An error has occurred: {e.Message}";
+                var msg = $"Task {taskContext.TaskId}, named {taskContext.TaskName} is not completed. An error has occurred: {e.Message}";
```

Comment:
For future: we have to change header of error emails to include there taskid (it will make emails more informative)